assets/scss/local.scss

```diff
@@ -7,7 +7,7 @@
   padding: 30px 15px;
 }
 
-#active-contacts, #inactive-contacts {
+#active-contacts, #inactive-contacts, #contact-search-results {
```

Comment:
should it just have a class like `.contact-list` rather than all the ids?

Reply:
contact-list class added